tox.ini

```diff
@@ -124,6 +129,11 @@ commands =
 
     manifest-checker: check-manifest --ignore "docs/_build*,docs/historic/*,admin*,bin/admin*,twisted/topfiles/*.Old"
 
+    pycodestyle: python {toxinidir}/admin/pycodestyle-twisted.py {posargs:admin src/twisted}
+    ; Travis-CI version is executing in diff mode without any input arguments.
+    cipycodestyle: /bin/sh -c "git diff trunk... | python {toxinidir}/admin/pycodestyle-twisted.py --diff"
```

Comment:
I suggest renaming this `pycodestyle-diff` as it will be useful outside of CI. When I run `tox -e pycodestyle` I get thousands of lines of output to sift through, so it's not much use on its own.

Additionally, for interactive use it would be useful to do `git diff trunk` instead of `git diff trunk...` so that uncommitted changes to tracked files are checked.